src/main/java/io/aiven/kafka/connect/common/config/AivenCommonConfig.java

```diff
@@ -82,6 +84,30 @@ protected static void addOutputFieldsFormatConfigGroup(final ConfigDef configDef
             FORMAT_OUTPUT_FIELDS_VALUE_ENCODING_CONFIG,
             FixedSetRecommender.ofSupportedValues(OutputFieldEncodingType.names())
         );
+
+        addFormatTypeConfig(configDef, formatGroupCounter);
```

Comment:
I'd put it before everything in this group as it's kind of most important config in `format.output.*`.

Reply:
- [ ] Move a config